### PR TITLE
Add new update UI

### PR DESCRIPTION
**Pull request by PeterAsenov22**

### CODE REVIEW

File: src/app/components/did/did-keys/did-keys.component.scss [hunk 1]
```diff
@@ -86,7 +86,13 @@ $mainColor: #2F5BE7;
...
-  width: calc(100% - 30px);
+  bottom: 0px;
+position: absolute;
+width: 100%;
+left: 0px;
+padding: 10px 15px;
+-webkit-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+-moz-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+background: #fff;
```
Comment (sanchopansa) [inline src/app/components/did/did-keys/did-keys.component.scss]: Minor: spacing is off.

File: src/app/components/did/management-keys/management-keys.component.scss [hunk 2]
```diff
@@ -87,7 +87,13 @@ $mainColor: #2F5BE7;
...
-  width: calc(100% - 30px);
+  bottom: 0px;
+position: absolute;
+width: 100%;
+left: 0px;
+padding: 10px 15px;
+-webkit-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+-moz-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+background: #fff;
```
Comment (sanchopansa) [inline src/app/components/did/management-keys/management-keys.component.scss]: Minor: spacing is off.

File: src/app/components/did/preview-did/preview-did.component.scss [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  background: #fff;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  z-index: 3;
+  padding: 15px 0px;
+  -webkit-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+-moz-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
```
Comment (sanchopansa) [inline src/app/components/did/preview-did/preview-did.component.scss]: Also here.

File: src/app/components/did/services/services.component.scss [hunk 4]
```diff
@@ -61,7 +61,13 @@ $mainColor: #2F5BE7;
...
-  width: calc(100% - 30px);
+  bottom: 0px;
+position: absolute;
+width: 100%;
+left: 0px;
+padding: 10px 15px;
+-webkit-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+-moz-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+background: #fff;
```
Comment (sanchopansa) [inline src/app/components/did/services/services.component.scss]: Here as well

File: src/app/components/did/summary/summary.component.scss [hunk 5]
```diff
@@ -8,7 +8,13 @@
...
-  width: calc(100% - 30px);
+  bottom: 0px;
+position: absolute;
+width: 100%;
+left: 0px;
+padding: 10px 15px;
+-webkit-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+-moz-box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+box-shadow: 0px -4px 5px 0px rgba(179,179,179,1);
+background: #fff;
```
Comment (sanchopansa) [inline src/app/components/did/summary/summary.component.scss]: And here

File: src/app/components/did/update-did/service-form/service-form.component.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      Service endpoint
+      <i class="fas fa-info-circle fa-lg text-primary" ngbTooltip="{{endpointTooltipMessage}}" tooltipClass="custom-tooltip"></i>
+    </label>
+    <input
+      type="text"
+      class="form-control input-md"
+      formControlName="endpoint"
+      id="endpoint"
+      placeholder="Service endpoint"
+      pattern="^(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$"
```
Comment (sanchopansa) [inline src/app/components/did/update-did/service-form/service-form.component.html]: A general comment: I'm wondering if it makes sense to use an existing library for validation of URLs (and maybe other data as well), such as this one https://validatejs.org/#validators-url, for example. Just something to consider, as usually these libraries have very well tested and robust validators and validating URLs in particular is tricky.

Comment (PeterAsenov22) [reply src/app/components/did/update-did/service-form/service-form.component.html]: I will consider it for the next PR.

**Pull request merged**